src/molecules/Modal/index.js

```diff
@@ -29,6 +29,26 @@ class Modal extends React.Component {
     }
   }
 
+  onAfterOpen = () => {
+    const { onAfterOpen } = this.props;
+
+    document.body.style.overflow = 'hidden';
+
+    if (onAfterOpen) {
+      onAfterOpen();
```

Comment:
Also, since this is part of the ReactModal api which is already exposed, why do we need to force this on the user, instead of just allowing them to pass it in in themselves?

If you want this to be the default behavior, I suggest considering a defaultProp declaration.

Reply:
@drewdrewthis not sure what you're getting at here? I'm still allowing the user to pass this in or not. If you look below, I'm not passing the `onAfterOpen` prop to React Modal, I'm now passing in this instance method instead, so I'm ensuring that the user's `onAfterOpen` prop is still called if provided